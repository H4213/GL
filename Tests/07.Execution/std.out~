Introduire la valeur pour la variable b : 
a*b : 125
e : 11
