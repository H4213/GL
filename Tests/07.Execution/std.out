a*b : 125
e : 11
